### PR TITLE
README update to make github page also reflect that Mainnet is launched

### PR DESCRIPTION
**Pull request by sesam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +19,7 @@ To learn more, read our [introduction to MimbleWimble and Grin](doc/intro.md).
 
 ## Status
 
-Grin is still an infant, much is left to be done and [contributions](CONTRIBUTING.md) are welcome (see below). Check our [mailing list archives](https://lists.launchpad.net/mimblewimble/) for the latest status.
+Grin is live with MainNet. Still, much is left to be done and [contributions](CONTRIBUTING.md) are welcome (see below). Check our [mailing list archives](https://lists.launchpad.net/mimblewimble/) for the latest status.
```
Comment (kargakis) [inline README.md]: lowercased "mainnet" is used it the website - we should probably follow that here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,22 +2,23 @@
 
 Find an area you can help with and do it. Open source is about collaboration and open participation. Try to make your code look like what already exists and submit a pull request.
 
-The [list of issues](https://github.com/mimblewimble/grin/issues) is a good place to start, especially the ones tagged as "help wanted" (but don't let that stop you from looking at others). If you're looking for additional ideas, the code includes `TODO` comments for minor to major improvements. Grep is your friend.
+The [list of issues](https://github.com/mimblewimble/grin/issues) is a good place to start, especially the ones tagged as "help wanted" (but don't let that stop you from looking at others). If you're looking for additional ideas, the code includes `TODO` comments for minor to major improvements. Use Search in files in your code editor, or `grep "TODO" -r src/`.
```
Comment (ignopeverell) [inline CONTRIBUTING.md]: Same grep remark as #2440

**Comment by ignopeverell** [comment]
Looks like there are overlapping chances with #2440 ?

**Comment by sesam** [comment]
Yes, good eye, I too see it now. Error between chair and keyboard. Duplicated commit force-pushed out of here.

**Pull request merged**